modules/escalation/main.tf

```
@@ -10,13 +10,9 @@ resource "opsgenie_escalation" "this" {
     notify_type = try(var.escalation.rule.notify_type, "default")
     delay       = try(var.escalation.rule.delay, 0)
 
-    dynamic "recipient" {
-      for_each = try(var.escalation.rule.recipients, [])
-
-      content {
-        id   = recipient.value.id
-        type = recipient.value.type
-      }
+    recipient {
+      type = recipient.value.type
+      id   = recipient.value.id
```

These are faililng tests since the dynamic was removed `recipient` is no longer a valid value.

I think you meant
```suggestion
      type = var.escalation.rule.recipient.type
      id   = var.escalation.rule.recipient.type
```

---

And there might need to be a variable change for this submodule